overlord/configstate/configcore/watchdog_test.go

```diff
@@ -46,16 +46,10 @@ var _ = Suite(&watchdogSuite{})
 func (s *watchdogSuite) SetUpTest(c *C) {
 	s.configcoreSuite.SetUpTest(c)
 
-	dirs.SetRootDir(c.MkDir())
 	s.mockEtcEnvironment = filepath.Join(dirs.SnapSystemdConfDir, "10-snapd-watchdog.conf")
-
 	s.systemctlArgs = nil
```

Comment:
This is done in configcoreSuite.SetUpTest() so we shoudl be able to remove it here.